trapper/metrics/metric.py

```diff
@@ -0,0 +1,82 @@
+from abc import ABCMeta, abstractmethod
```

Comment:
I think it would be nice if there was a way to see the available metrics and their arguments without looking at jury's source code although I guess it is hard to do. Actually, with the datasets integration, we have a similar problem (kind of) but they have more than 100 datasets so it is OK for that. In the optimizers, model_wrappers etc, we have our wrappers sitting as a layer between trapper and allennlp (optimizers) and transformers (model_wrappers). What I would like to have a similar layer for the metrics. The problem is that Jury already has a nice class hierarchy with concrete subclasses as well as factories, therefore I do not like repeating the whole structure here in trapper. Nevertheless, I wanted to share this thought.

Reply:
I'll look into those implementations first.